lib/rubocop/cop/rspec/shared_examples.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Enforces use of string to titleize shared examples.
+      #
+      # @example
+      #   # bad
+      #   it_behaves_like :foo_bar_baz
+      #   it_should_behave_like :foo_bar_baz
+      #   shared_examples :foo_bar_baz
+      #   shared_examples_for :foo_bar_baz
+      #   include_examples :foo_bar_baz
+      #
+      #   # good
+      #   it_behaves_like 'foo bar baz'
+      #   it_should_behave_like 'foo bar baz'
+      #   shared_examples 'foo bar baz'
+      #   shared_examples_for 'foo bar baz'
+      #   include_examples 'foo bar baz'
+      #
+      class SharedExamples < Cop
+        def_node_matcher :shared_examples, <<-PATTERN
+          (send nil? {#{(SharedGroups::ALL + Includes::ALL).node_pattern}} $!str ...)
+        PATTERN
+
+        def on_send(node)
+          shared_examples(node) do |ast_node|
+            checker = Checker.new(ast_node)
+            return unless checker.offensive?
```

Comment:
This block is only executed when an `ast_node` is found that matches the criteria `!str`. This means `ast_node.str_type?` will always be false, and `checker.offensive?` will always be true.

Reply:
Right, I got it now !